source/retryable-writes/tests/README.rst

```diff
@@ -175,6 +175,26 @@ Each YAML file has the following keys:
       - ``data``: The data that should exist in the collection after the
         operation has been run.
 
+Split Batch Tests
+=================
+
+The YAML tests specify bulk write operations that are split by command type
+(e.g. sequence of insert, update, and delete commands). Multi-statement write
+operations may also be split due to ``maxWriteBatchSize`` or
+``maxBsonObjectSize``. For instance, an insertMany operation with three 9 MB
+documents would be split into three insert commands in order to respect the 16
+MB ``maxBsonObjectSize`` limit. Drivers should manually test that operations
+split due to ``maxWriteBatchSize`` and ``maxBsonObjectSize`` can still be
+retried.
+
+The ``onPrimaryTransactionalWrite`` fail point's ``skip`` option may be used to
```

Comment:
You may want to add a reminder that after the skip the mode will transition to always on and no subsequent writes will succeed.

Reply:
Will do.